project_tracking/db_action.py

```diff
@@ -521,7 +521,7 @@ def create_project(project_name, fms_id=None, session=None):
     return session.scalars(select(Project).where(Project.name == project_name)).one()
 
 
-def ingest_run_processing(project_id, ingest_data, session=None):
+def ingest_run_processing(project_id: str, ingest_data, session=None):
```

Comment:
project_id should be a str or an int? This would have implication in many routes signature

Reply:
How do we want IDs being int jsonified so string to be?